apps/api-extractor-model/config/jest.json

```diff
@@ -1,3 +1,3 @@
 {
-  "isEnabled": true
+  "isEnabled": false
```

Comment:
Apparently our Jest config still annoyingly requires at least one unit test.  We should fix that.  The old Mocha task doesn't have this problem.

Reply:
Yeah. We should really refactor the whole jest task to just use the standalone executable, like the way we invoke tsc.

Reply:
+1